x-pack/plugins/secops/server/graphql/network/schema.test.ts

```diff
@@ -99,16 +104,87 @@ const testCaseSource = {
   },
 };
 
-describe('Test Source Schema', () => {
+const testNetworkDnsSource = {
+  id: 'Test case to query Network DNS Domains',
+  query: `
+    query GetNetworkDnsQuery(
+      $sort: NetworkDnsSortField!
+      $isPtrIncluded: Boolean!
+      $timerange: TimerangeInput!
+      $pagination: PaginationInput!
+      $filterQuery: String
+    ) {
+      source(id: "default") {
+        NetworkDns(
+          isPtrIncluded: $isPtrIncluded
+          sort: $sort
+          timerange: $timerange
+          pagination: $pagination
+          filterQuery: $filterQuery
+        ) {
+          totalCount
+          edges {
+            node {
+              _id
+              dnsBytesIn
+              dnsBytesOut
+              dnsName
+              queryCount
+              uniqueDomains
+            }
+            cursor {
+              value
+            }
+          }
+          pageInfo {
+            endCursor {
+              value
+            }
+            hasNextPage
+          }
+        }
+      }
+    }
+	`,
+  variables: {
+    timerange: {
+      interval: '12h',
+      to: 1514782800000,
```

Comment:
As in other comments of date time ranges, I would use the human form and do a `.valueOf()` to make it readable for when the tests break we know what the date time ranges are.